include/hardware.h

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (C) 2002-2021  The DOSBox Team
+ *  Copyright (C) 2002-2023  The DOSBox Team
```

Comment:
We are keeping "DOSBox Team" copyrights intact; in cases like this one, add another copyright line for "DOSBox Staging Team" (grep through the code will give plenty of examples).

Reply:
Yeah okay. TBH I did not even read it, just updated the date 😄

Reply:
Done in all files.